addcomplete.php

```diff
@@ -0,0 +1,40 @@
+<?php
+require_once 'head.html';
+require_once 'Tree.php';
+require_once 'TreeDBConnect.php';
+
+$tree = new TreeDBConnect();
+
+
+if ($_SERVER["REQUEST_METHOD"] == "POST") {
```

Comment:
I would also do an `isset($_POST['name'])` here to check that not only do we have a post request, it's the right post request. 


VERY niche but worth doing

Reply:
I've added the above suggestion - I'd like to chat to you when you have a second to go through the actual usefulness of it?